client/script/client.js

```diff
@@ -1,4 +1,5 @@
 var SESSION_KEY_ERROR = 'Invalid session number: must be 32 character combination of letters and numbers';
+var SESSION_KEY_ERROR = 'Invalid participation code: must be 32 character combination of letters and numbers';
```

Comment:
When we switch to base32 a whole other bunch of error messages and regexs must be switched. I prefer to do that as part of the switch to make sure we do not miss anything.